cluster/addons/OWNERS

```diff
@@ -1,5 +1,8 @@
 # See the OWNERS docs at https://go.k8s.io/owners
 
+approvers:
+- sig-architecture-approvers
```

Comment:
I would have expected this to fall up to clusters/OWNERS

Reply:
ack moving this one level up